src/consensus/params.h

```diff
@@ -51,6 +51,8 @@ struct Params {
     int uahfHeight;
     /** Block height at which OP_RETURN replay protection stops */
     int antiReplayOpReturnSunsetHeight;
+    /** Block height at which TVS43 becomes active */
+    int TVS43Height;
```

Comment:
TVS43 ?
Maybe a tiny addition on https://github.com/Bitcoin-Clashic/tvs ?

Reply:
Work in progress..
https://github.com/clashicly/tvs/blob/patch-1/tvs-0043.mediawiki